deployment/common/changeset/capability_registry_test.go

```diff
@@ -0,0 +1,55 @@
+package changeset
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zapcore"
+
+	chainsel "github.com/smartcontractkit/chain-selectors"
+
+	"github.com/smartcontractkit/chainlink/deployment/common/view/v1_0"
+	"github.com/smartcontractkit/chainlink/deployment/environment/memory"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/capabilities_registry"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+func TestHydrateCapabilityRegistry(t *testing.T) {
+	b, err := os.ReadFile("testdata/capability_registry_view.json")
+	require.NoError(t, err)
+	require.NotEmpty(t, b)
+	var capabilityRegistryView v1_0.CapabilityRegistryView
+	require.NoError(t, json.Unmarshal(b, &capabilityRegistryView))
+
+	chainID := chainsel.TEST_90000001.EvmChainID + 1
```

Comment:
why `+ 1`?
i think there are multiple TEST selectors if you simply want one that is not 90000001

Reply:
Used +1 based on how chains are generated [here](https://github.com/smartcontractkit/chainlink/blob/f3ae32858fced94cb3438dd4e11cf33b4fb44dd7/deployment/environment/memory/chain.go#L48).